src/textures/CompressedCubeTexture.js

```diff
@@ -0,0 +1,19 @@
+import { CubeReflectionMapping } from '../constants.js';
+import { CompressedTexture } from './CompressedTexture.js';
+
+class CompressedCubeTexture extends CompressedTexture {
+
+	constructor( images, format, type ) {
+
+		super( undefined, images[ 0 ].width, images[ 0 ].height, format, type, CubeReflectionMapping );
```

Comment:
@donmccurdy Would it work to pass in `[]` for `mipmaps` instead? I'm working on the types for this. If taken strictly, `Texture.mipmaps` and `CompressedTexture.mipmaps` would now need to be `ImageData[] | undefined` instead of `ImageData[]`, which is more disruptive of a change for the types.

Reply:
I believe it's correct that the `.mipmaps` property is a non-nullable array, but *not* safe to pass an empty array up to the constructor here... I would just assume that mipmaps gets populated correctly during texture initialization, and not make the property nullable in the TS definitions.

Reply:
Okay, thanks, good to know. For my own education, where do the `mipmaps` end up getting set for the `CompressedCubeTexture` in `KTX2Loader`?

Reply:
You can see where it would be set in https://github.com/mrdoob/three.js/pull/26642, but  I can't promise that KTX2Loader sets up all texture types correctly today. Constructing compressed array, cube, and 3D textures is not a fully-documented API, and we don't have many test cases for it, so I'm kind of trial-and-erroring my through this part.